README.md

```diff
@@ -32,7 +32,8 @@ This package should only be used in projects starting from scratch, since it ove
     ```python
     KEYCLOAK_CONFIG = {
         'SERVER_URL': '<PUBLIC_SERVER_URL>',
-        'INTERNAL_URL': '<INTERNAL_SERVER_URL>',
+        'INTERNAL_URL': '<INTERNAL_SERVER_URL>', # Optional: Default is SERVER_URL
+        'BASE_URL': '', # Optional: Default matches Keycloak's default '/auth'
```

Comment:
It should be `'BASE_PATH'`, no?
I'd also add to the comment what it's meant for:

To customise Keycloak's URL path from `SERVER_URL/auth/admin/...` or `SERVER_URL/auth/realms/...` to `SERVER_URL/some_path_you_prefer/admin/...` and `SERVER_URL/some_path_you_prefer/realms/...`

Reply:
Thanks for catching that. I put a bit more description in a separate heading